packages/suite/src/utils/suite/analytics.ts

```diff
@@ -42,7 +43,7 @@ export const getSuiteReadyPayload = (state: AppState) => ({
     screenWidth: getScreenWidth(),
     screenHeight: getScreenHeight(),
     platformLanguages: getPlatformLanguages().join(','),
-    tor: state.suite.tor,
+    tor: getIsTorEnabled(state.suite.torStatus),
```

Comment:
Ah, I just realized that `tor` attribute in `suite-ready` is probably not fully reliable

Reply:
Could you explain?

Reply:
It was just a note for me.